modules/extension/xsd/xsd-wcs/src/main/resources/org/geotools/wcs/v1_1/gml4wcs.xsd

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schema targetNamespace="http://www.opengis.net/gml" 
+xmlns:gml="http://www.opengis.net/gml"
+xmlns="http://www.w3.org/2001/XMLSchema" 
+elementFormDefault="qualified" version="1.1.1">
+	<annotation>
+		<appinfo source="urn:opengis:specification:gml:schema-xsd:gml:3.1.1">gml.xsd</appinfo>
+		<documentation>GML profile for WCS 1.2. Primary editor: Arliss Whiteside. Last updated 2007-06-06
+		Copyright © 2007 Open Geospatial Consortium, Inc. All Rights Reserved. </documentation>
+	</annotation>
```

Comment:
FYI, this is actually not mentioned in the developer guide. Some quick google seaches show that the only place an IP check is mentioned is http://docs.geotools.org/latest/developer/procedures/check.html, and even there the IP check process is not described.